homeassistant/components/uptime_kuma/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""The Uptime Kuma integration."""
+from datetime import timedelta
+import logging
+
+import async_timeout
+from uptime_kuma_monitor import UptimeKumaMonitor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["binary_sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Uptime Kuma integration."""
+
+    async def async_get_uptime_kuma_data():
+        with async_timeout.timeout(10):
+            utkm = await hass.async_add_executor_job(
+                UptimeKumaMonitor,
+                f"{entry.data[CONF_HOST]}:{entry.data[CONF_PORT]}",
+                entry.data[CONF_USERNAME],
+                entry.data[CONF_PASSWORD],
+                entry.data[CONF_VERIFY_SSL],
+            )
+            return utkm.data
+
+    coordinator = update_coordinator.DataUpdateCoordinator(
+        hass,
+        logging.getLogger(__name__),
```

Comment:
Create a constant for this:
`LOGGER = logging.getLogger(__name__)`
```suggestion
        LOGGER,
```